content/en/docs/setup/best-practices/certificates.md

```diff
@@ -19,16 +19,23 @@ This page explains the certificates that your cluster requires.
 
 ## How certificates are used by your cluster
 
-Kubernetes requires PKI for the following operations:
-
-* Client certificates for the kubelet to authenticate to the API server
-* Server certificate for the API server endpoint
-* Client certificates for administrators of the cluster to authenticate to the API server
-* Client certificates for the API server to talk to the kubelets
-* Client certificate for the API server to talk to etcd
-* Client certificate/kubeconfig for the controller manager to talk to the API server
-* Client certificate/kubeconfig for the scheduler to talk to the API server.
-* Client and server certificates for the [front-proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
+Kubernetes requires certificates to allow the following authentication to take place:
+
+### Client certificates authenticate ...
+* the kubelet to the API server
+* cluster administrator users to the API server
+* the API server to the kubelets
+* the API server to etcd
+
+### Server certificate authenticates ...
+* the API server endpoint to connecting clients
```

Comment:
A Kubernetes cluster has more than one server certificate:
- etcd has an X.509 certificate that it presents to clients
- the kube-apiserver has an X.509 certificate
- the kubelet can serve HTTPS and uses an X.509 certificate for that
- APIServices (aggregated API servers) present server certificates
- if the API server calls HTTP services (“webhooks”), for example during admission, then the called service can present a certificate too.

We should list some or all of these, and IMO we should avoid using the singular for “server”.